### PR TITLE
Create FC-Virus

### PR DESCRIPTION
**Pull request by qdu-bioinfo**
### Title
Update: FC-Virus to version 1.0.0

### Summary
This PR aims to update the FC-Virus recipe to version 1.0.0. This update includes improvements such as:

Enhanced Algorithm: The core algorithm has been optimized for better performance.
New Visualization Tools: Added new visualization tools for data analysis.

### Changes Made
Updated meta.yaml file to reflect the new version and dependencies.

### Dependencies
Added dependency on gcc.

### Testing
Ran all unit tests and integration tests, all of which passed successfully.
Confirmed functionality against sample datasets.

Commands
Please apply the following command to help manage this PR:

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by qdu-bioinfo** [comment]
@antunderwood

**Comment by qdu-bioinfo** [comment]
@martin-g  ,Can you help review this PR?

**Comment by qdu-bioinfo** [comment]
@BiocondaBot please add label

File: recipes/FC-Virus/mete.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package:
+  name: FC-Virus
+  version: "1.0.0"
+  noarch: generic
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:4]: This should be moved under 'build'

File: recipes/FC-Virus/mete.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+source:
+  url: https://github.com/qdu-bioinfo/FC-Virus/archive/refs/heads/main.tar.gz
+  git_tag: main
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:8]: Please remove this!
It is ignored now anyway. It should be used with 'git_url'

File: recipes/FC-Virus/mete.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  sha256: a5c91ea315dc7b64b75986128d97c08522aae332cb640fd2c115cc379b4adc51
+
+build:
+  number: 0
+  skip: true
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:13]: This means that the recipe is not built for all platforms

File: recipes/FC-Virus/mete.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+requirements:
+  build:
+    - gcc
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:17]:
```suggestion
    - {{ compiler("c") }}
```

File: recipes/FC-Virus/mete.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    - gxx_linux-64
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:18]:
```suggestion
    - {{ compiler("cxx") }}
```

File: recipes/FC-Virus/mete.yaml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    - make
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:19]:
```suggestion
    - make
  host:
```

File: recipes/FC-Virus/mete.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    - boost
+  run:
+    - boost
+
+about:
+  home: https://github.com/qdu-bioinfo/FC-Virus
+  license: MIT
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:26]: Please add 'license_file: ...' too

**Review by martin-g — CHANGES_REQUESTED** [review]

File: recipes/FC-Virus/mete.yaml (hunk 1 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thanks for your reply. I've moved the relevant section under 'build' as suggested.

File: recipes/FC-Virus/mete.yaml (hunk 2 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thanks for your reply. I have removed that section as you suggested.

File: recipes/FC-Virus/mete.yaml (hunk 3 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thanks for your reply. I have removed that section.

File: recipes/FC-Virus/mete.yaml (hunk 7 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: I have added the 'license_file: ...' section as requested. Thank you!

File: recipes/FC-Virus/mete.yaml (hunk 5 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thank you for your reply. I have completed the modifications.

File: recipes/FC-Virus/mete.yaml (hunk 6 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml:12]: Thank you for your reply. I have completed the modifications.

File: recipes/FC-Virus/mete.yaml (hunk 4 above)
Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thank you for your reply. I have completed the modifications.

File: recipes/FC-Virus/mete.yaml [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    - {{ compiler("cxx") }}
+    - make
+  host:
+    - boost
+  run:
+    - boost
+    
+about:
+  home: https://github.com/qdu-bioinfo/FC-Virus
+  license: MIT
```
Comment (qdu-bioinfo) [inline recipes/FC-Virus/mete.yaml:19]: -  license_file: https://github.com/qdu-bioinfo/FC-Virus/LICENSE
+  license_file: LICENSE

File: recipes/FC-Virus/mete.yaml [hunk 9]
```diff
@@ -0,0 +1,32 @@
+package:
```
Comment (martin-g) [inline recipes/FC-Virus/mete.yaml:1]: @qdu-bioinfo  Please rename the file from `mete.yaml` to `meta.yaml`

Comment (qdu-bioinfo) [reply recipes/FC-Virus/mete.yaml]: Thank you for your reply, I have renamed it.

**Comment by qdu-bioinfo** [comment]
Due to my package name being capitalized, there was an error in the "Linux Tests" section. I may have to abandon this pull_dequest and change the package name before attempting to pull it again.

**Comment by qdu-bioinfo** [comment]
@BiocondaBot Do you know the reason why my test files are not available in the "/test_tmp" directory during testing?

**Comment by qdu-bioinfo** [comment]
@martin-g @BiocondaBot Do you know the reason why my test files are not available in the "/test_tmp" directory during testing? Thanks.

File: recipes/fc-virus/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  run_exports:
+    - {{ pin_subpackage("fc-virus", max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - make
+  host:
+    - make
```
[the remark this answers is not in the document]
Comment (qdu-bioinfo) [reply recipes/fc-virus/meta.yaml:20]: Do you know the reason why my test files are not available in the "/test_tmp" directory during testing?

**Comment by qdu-bioinfo** [comment]
@BiocondaBot add label: bug

**Pull request closed without merging**